src/index.ts

```diff
@@ -57,30 +57,34 @@ class MongooseTsgen extends Command {
   static args = [{ name: "model_path" }];
 
   private getConfig() {
-    const { flags: cliFlags, args } = this.parse(MongooseTsgen);
-
-    type FlagConfig = Omit<typeof cliFlags, "config" | "output" | "project"> & {
-      output: string;
-      project: string;
-    };
-
-    const configFileFlags: Partial<FlagConfig> = paths.getConfigFromFile(cliFlags.config);
-
-    // remove "config" since its only used to grab the config file
-    delete cliFlags.config;
-
-    // we cant set flags as `default` using the official oclif method since the defaults would overwrite flags provided in the config file.
-    // instead, well just set "output" and "project" as default manually if theyre still missing after merge with configFile.
-    configFileFlags.output = configFileFlags.output ?? "./src/interfaces";
-    configFileFlags.project = configFileFlags.project ?? "./";
-
-    return {
-      flags: {
-        ...configFileFlags,
-        ...cliFlags
-      } as FlagConfig,
-      args
-    };
+    try {
+      const { flags: cliFlags, args } = this.parse(MongooseTsgen);
+
+      type FlagConfig = Omit<typeof cliFlags, "config" | "output" | "project"> & {
+        output: string;
+        project: string;
+      };
+
+      const configFileFlags: Partial<FlagConfig> = paths.getConfigFromFile(cliFlags.config);
+
+      // remove "config" since its only used to grab the config file
+      delete cliFlags.config;
+
+      // we cant set flags as `default` using the official oclif method since the defaults would overwrite flags provided in the config file.
+      // instead, well just set "output" and "project" as default manually if theyre still missing after merge with configFile.
+      configFileFlags.output = configFileFlags?.output ?? "./src/interfaces";
+      configFileFlags.project = configFileFlags?.project ?? "./";
+
+      return {
+        flags: {
+          ...configFileFlags,
+          ...cliFlags
+        } as FlagConfig,
+        args
+      };
+    } catch (error: any) {
+      throw new Error(error.message);
```

Comment:
What's the purpose for this change?

Reply:
I made this change because error.stack did not appear (even declaring the **-d** option)

I had to add this catch to catch the error.

without that I only had the generic information

```SyntaxError: Unexpected token } in JSON at position 683```

not knowing where was the problem with parsing the JSON